app/services/users.js

```diff
@@ -42,5 +42,14 @@ module.exports.signIn = body =>
         throw errors.invalidCredentials('Invalid credentials, please try again');
       }
 
-      return jwt.encode(user.email, process.env.SECRET_KEY);
+      return models.session.findOne({ where: { user_id: user.id } }).then(previousSession => {
```

Comment:
`user_id` 👀 
Sequelize can handle searches for both the original param name and the alias, so this still works, but please use the alias, so we can keep our code in `camelCase`